config/environments/development.rb

```diff
@@ -85,4 +85,16 @@
     # Add warnings directly to the Rails log
     Bullet.rails_logger = true
   end
+
+  # Mailer configuration
+  config.action_mailer.smtp_settings = {
+    address: ENV['EMAIL_ADDRESS'],
```

Comment:
I don't like the naming too much, `MAIL_SERVER_ADDRESS` `MAIL_SERVER_USERNAME` and `MAIL_SERVER_PASSWORD` seem more fitting